src/v2/config_service_v2_client.js

```diff
@@ -19,7 +19,7 @@ const gax = require('google-gax');
 const merge = require('lodash.merge');
 const path = require('path');
 
-const VERSION = require('../../package.json').version;
+const VERSION = require('../../../package.json').version;
```

Comment:
This is auto-generated code :(  It would get replaced the next time we run autosynth.  One option here would be to write some code in synth.py that replaces this line at generation time.  @crwilcox and @kinwa91 would be able to point to a few examples here :)

Reply:
Ideally this code should use [`pkgUp`](https://www.npmjs.com/package/pkg-up) rather than relative paths.

Reply:
Done.